pkg/instancetype/instancetype.go

```diff
@@ -893,6 +897,9 @@ func applyCPU(field *k8sfield.Path, instancetypeSpec *instancetypev1beta1.Virtua
 		vmiSpec.Domain.CPU.Sockets = instancetypeSpec.CPU.Guest
 	case instancetypev1beta1.PreferThreads:
 		vmiSpec.Domain.CPU.Threads = instancetypeSpec.CPU.Guest
+	case instancetypev1beta1.PreferSpread:
+		vmiSpec.Domain.CPU.Sockets = instancetypeSpec.CPU.Guest / 2
```

Comment:
`2` should also be a const defined somewhere near `PreferSpread` in the API definition.

Reply:
I have defined the constant just above the switch statement, but if you still prefer to move it into the `type.go` file, I can do that.